compose.yaml

```
@@ -0,0 +1,9 @@
+services:
+  db:
+    image: "postgis/postgis:15-3.4-alpine"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=zoning
```

I saw the readme already mentions creating an .env file from the sample.env. In which case, I think it makes sense to really lean into using the .env as the source.

```suggestion
      - POSTGRES_USER=${DATABASE_USER}
      - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
      - POSTGRES_DB=${DATABASE_NAME}
```